StreamChatSwiftUITests/Tests/ChatChannel/MessageListViewAvatars_Tests.swift

```diff
@@ -101,7 +105,7 @@ class MessageListViewAvatars_Tests: StreamChatTestCase {
             onScrollToBottom: {},
             onLongPress: { _ in }
         )
-
-        return messageListView
+        .applyDefaultSize()
+        .environmentObject(ChatChannelViewModel(channelController: channelController))
```

Comment:
Add to update a lot of tests, and there is still more tests to update because of the `@EnvironmentObject`. So another advantage of `@Environment` is that it won't be required to update the tests since it will fallback to the previous implementation.